public/assets/sass/style.scss

```diff
@@ -87,8 +120,30 @@ body {
   flex-wrap: nowrap;
   font-family: Roboto, "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
-  margin-left: 5px;
-  margin-right: 5px;
+  margin-left: 10px;
+  margin-right: 10px;
+  overflow: hidden;
+  position: relative;
+
+  .right-side {
```

Comment:
I'm not sure if this name is very clear. We need to think of a different name for this.

maybe `float-right`?

tell me what you think